frontend/components/tableDisplay/petTableDisplay.tsx

```diff
@@ -0,0 +1,126 @@
+import { Table, Input, Button, Space } from 'antd';
+import { PlusOutlined, ImportOutlined } from '@ant-design/icons';
+import SwitchTag from './switchTag';
+import TableName from './tableNameColumn';
+import React, { useState } from 'react';
+
+const PetTableDisplay = () => {
+  const [searchText, setSearchText] = useState("");
+  const [searchedColumn, setSearchedColumn] = useState("");
+
+  const { Search } = Input;
+  const onSearch = () => { };
+
+
+  const columns = [
+    {
+      title: 'ID',
+      dataIndex: 'key',
+      defaultSortOrder: 'ascend',
+      sorter: (a, b) => a.key - b.key,
+    },
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      sorter: (a, b) => a.name.localeCompare(b.name),
```

Comment:
any reason we require `localeCompare`?

Reply:
This is the method in the antd documentation to create a sorting system in the column, but happy to change if you think there's a better way

Reply:
Stick with `localeCompare`. It correctly compare the values if it's non-English (if I remember correctly).